src/main/resources/add-ons-to-index.json

```diff
@@ -2871,6 +2902,7 @@
       "status": "INACTIVE",
       "type": "OMOD",
       "name": "Quartz Scheduler",
+      "icon": "calendar",
```

Comment:
This one isn't really relevant so I guess this should be replaced/ deleted.Check https://wiki.openmrs.org/display/docs/Quartz+Scheduler+Module for more info